SPECS/forkexecd.spec

```diff
@@ -16,19 +17,19 @@ BuildRequires:  ocaml-stdext-devel
 BuildRequires:  ocaml-uuidm-devel
 BuildRequires:  ocaml-xcp-idl-devel
 BuildRequires:  ocaml-oclock-devel
-#Requires:  redhat-lsb-core
+BuildRequires:  ocaml-systemd-devel
+BuildRequires:  systemd-devel
```

Comment:
If you add `BuildRequires:  ocaml-systemd-devel`, which depends on `systemd-devel`, do you then still need `BuildRequires:  systemd-devel`?